hazelcast/proxy/multi_map.py

```diff
@@ -445,6 +449,47 @@ def put(self, key: KeyType, value: ValueType) -> Future[bool]:
         request = multi_map_put_codec.encode_request(self.name, key_data, value_data, thread_id())
         return self._invoke_on_key(request, key_data, multi_map_put_codec.decode_response)
 
+    def put_all(self, multimap: typing.Dict[KeyType, typing.Sequence[ValueType]]) -> Future[None]:
+        """Stores the given Map in the MultiMap.
+
+        The results of concurrently mutating the given map are undefined.
+        No atomicity guarantees are given. It could be that in case of failure some of the key/value-pairs get written, while others are not.
+
+        Warning:
+            This method uses ``__hash__`` and ``__eq__`` methods of binary form
+            of the key, not the actual implementations of ``__hash__`` and
+            ``__eq__`` defined in key's class.
+
+        Args:
+            multimap: the map corresponds to multimap entries.
+        """
+        check_not_none(multimap, "multimap can't be None")
+        if not multimap:
+            return ImmediateFuture(None)
+
+        partition_service = self._context.partition_service
+        partition_map: typing.DefaultDict[int, typing.List[typing.Tuple[Data, typing.List[Data]]]] = defaultdict(list)
+
+        for key, values in multimap.items():
+            try:
+                check_not_none(values, "values can't be None")
+                serialized_key = self._to_data(key)
+                serialized_values = []
+                for value in values:
+                    check_not_none(value, "value can't be None")
+                    serialized_values.append(self._to_data(value))
+                partition_id = partition_service.get_partition_id(serialized_key)
+                partition_map[partition_id].append((serialized_key, serialized_values))
+            except SchemaNotReplicatedError as e:
+                return self._send_schema_and_retry(e, self.put_all, multimap)
+
+        futures = []
+        for partition_id, entry_list in partition_map.items():
+            request = multi_map_put_all_codec.encode_request(self.name, entry_list)
+            future = self._invoke_on_partition(request, partition_id)
+            futures.append(future)
+        return combine_futures(futures).continue_with(lambda *args: None)
```

Comment:
```suggestion
        return combine_futures(futures).continue_with(lambda _: None)
```
I think this should also work, as we don't care about the returned list. No need to bind that to *args